packages/edit-site/src/components/global-styles/screen-root.js

```diff
@@ -44,16 +44,20 @@ function ScreenRoot() {
 						</CardMedia>
 					</Card>
 					{ !! variations?.length && (
-						<NavigationButton path="/variations">
-							<HStack justify="space-between">
-								<FlexItem>{ __( 'Browse styles' ) }</FlexItem>
-								<IconWithCurrentColor
-									icon={
-										isRTL() ? chevronLeft : chevronRight
-									}
-								/>
-							</HStack>
-						</NavigationButton>
+						<ItemGroup>
+							<NavigationButtonAsItem path="/variations">
```

Comment:
This "fixes" the semantic issue, but wrapping this single item in an `ItemGroup` is not ideal. Ideally I would like to use a vanilla `NavigatorButton` here instead of the `Item`-wrapped version, but unfortunately there is a styling inconsistency (#40591) that makes that weird too.

I think we can go with this bandaid fix, since we'll probably overhaul this view (#40478) anyway.

Reply:
What about overriding the `role`, so that at least we don't have a semantically weird one-item list ?

```suggestion
						<ItemGroup role={ null }>
							<NavigationButtonAsItem
								path="/variations"
								role={ null }
							>
```

Just a caveat: eslint complains about passing `null` as a value, but it's the only way to properly override this value (passing `undefined` would make `ItemGroup` and `Item` fallback on their default `role` values)

Reply:
Yeah, I'm hesitant to do that too since I agree with eslint that `role={ null }` is not at all kosher 😅 And neither is [`role="generic"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/generic_role), apparently.

My preference would be to fix the [focus outline inconsistency](https://github.com/WordPress/gutenberg/issues/40591) so we can just use a vanilla `NavigatorButton`. (A [potential path](https://github.com/WordPress/gutenberg/pull/40588#discussion_r858496317) to overriding semantics cleanly is the `as` prop though.)

Reply:
Agreed. Let's improve the focus outline inconsistency, and then look at a way to rewrite `ItemGroup` in a way that makes possible using `as` to override semantics (together with [a few other tweaks](https://github.com/WordPress/gutenberg/issues/34709) that we took note of)